src/lib/dialog/dialog.spec.ts

```diff
@@ -965,6 +973,73 @@ describe('MatDialog', () => {
       expect(container.hasAttribute('aria-labelledby')).toBe(false);
     }));
   });
+
+  describe('global dialog options', () => {
+    function createTestComponent(openConfig: MatDialogConfig, globalConfig: DialogGlobalOptions) {
+      // Reset the previously configured testing module to be able set new providers.
+      // The testing module has been initialized in the root describe group for the dialogs.
+      TestBed.resetTestingModule();
+      TestBed.configureTestingModule({
+        imports: [MatDialogModule],
+        declarations: [PizzaMsg],
+        providers: [{ provide: MAT_DIALOG_GLOBAL_OPTIONS, useValue: globalConfig }]
+      });
+
+      dialog.open(PizzaMsg, {
```

Comment:
You should move this into the individual tests. It's easier to follow, compared to passing in two arguments that look identical into a function.